### ISSUE TITLE
A11y: noninteractive element cannot have positive tabIndex value when using tabindex="0"

### ISSUE THREAD

**Issue by charbelnicolas** [sveltejs/vite-plugin-svelte#465]
### Describe the bug

I don't really know where to post this bug but I keep getting the same warning when using tabindex="0":

![20221015_104155](https://user-images.githubusercontent.com/14960414/195995347-aebf4155-0001-45df-9528-24a331cfdc9b.png)

![20221015_104249](https://user-images.githubusercontent.com/14960414/195995366-e9637ade-a41e-4450-ad1e-84f5934bbbc2.png)

If you know where I should report this bug, let me know!

### Reproduction

Add a tabindex="0" to any div element and see for yourself :)

### System Info

```shell
System:
    OS: Linux 6.0 Arch Linux
    CPU: (8) x64 Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
    Memory: 29.44 GB / 31.31 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 18.11.0 - /usr/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 8.19.2 - /usr/bin/npm
  Browsers:
    Chromium: 106.0.5249.119
```

### Severity

annoyance
